.gitignore

```diff
@@ -1,52 +1,143 @@
-# Temporary Python files
```

Comment:
Within the commit description:
s/endpoind/endpoint/

s/to constants routes and domain register/to the constant route & domain arrays/